src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -655,4 +656,19 @@ BOOST_AUTO_TEST_CASE(MishFunctionTest)
   CheckDerivativeCorrect<MishFunction>(desiredActivations,
                                         desiredDerivatives);
 }
+
+/**
+ * Basic test of the SQNL function.
+ */
+BOOST_AUTO_TEST_CASE(SQNLFunctionTest)
+{
+  const arma::colvec desiredActivations("-1 1 1 -1 0.75 -0.75 1 0");
+
```

Comment:
Are these values also hand calculated?

Reply:
Yes, this function is apparently very simple.